Traceback (most recent call last):
  File "/home/infratest/pytest/boto3/insstats.py", line 9, in <module>
    instances = [i for i in boto3.resource('ec2', region_name='ap-northeast-2').instances.all()]
  File "/home/infratest/pytest/boto3/insstats.py", line 9, in <listcomp>
    instances = [i for i in boto3.resource('ec2', region_name='ap-northeast-2').instances.all()]
  File "/usr/local/lib/python3.7/site-packages/boto3/resources/collection.py", line 83, in __iter__
    for page in self.pages():
  File "/usr/local/lib/python3.7/site-packages/boto3/resources/collection.py", line 166, in pages
    for page in pages:
  File "/usr/local/lib/python3.7/site-packages/botocore/paginate.py", line 255, in __iter__
    response = self._make_request(current_kwargs)
  File "/usr/local/lib/python3.7/site-packages/botocore/paginate.py", line 332, in _make_request
    return self._method(**current_kwargs)
  File "/usr/local/lib/python3.7/site-packages/botocore/client.py", line 276, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/usr/local/lib/python3.7/site-packages/botocore/client.py", line 586, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (UnauthorizedOperation) when calling the DescribeInstances operation: You are not authorized to perform this operation.
